src/System.IO.Packaging/src/System/IO/Packaging/PartBasedPackageProperties.cs

```diff
@@ -397,8 +397,8 @@ private void RecordNewBinding(PackageXmlEnum propertyenum, object value, bool in
                 // If the binding is an assignment rather than an initialization, set the dirty flag.
                 _dirty = !initializing;
             }
-            // Case of an initial value being set for a property.
-            else
+            // Case of an initial value being set for a property. If value is null, no need to do anything
+            else if (value != null)
```

Comment:
netfx throws an exception in this case if _package.InStreamingCreation is true... is that something we need to be concerned with in corefx?

Reply:
There is no property `Package.InStreamingCreation` and no reference to anything `Streaming`, so I'm assuming we don't need to include that in corefx.

Reply:
Ok, thanks.